src/Layers/xrRenderGL/glDetailManager_VS.cpp

```diff
@@ -148,6 +148,8 @@ void CDetailManager::hw_Render_dump(CBackend& cmd_list, const Fvector4& consts,
                 VERIFY(c_storage);*/
 
                 u32 dwBatch = 0;
+                std::vector<glm::vec4> uniformBuffer;
```

Comment:
It should be xr_vector.
```suggestion
                xr_vector<glm::vec4> uniformBuffer;
```

Reply:
as a newcomer/noob  I have no clue what is xr_vector, but i need a vector

Reply:
Just replace std::vector with xr_vector. They are the same, except that the latter uses custom allocator.